pkg/observer/observer_windows.go

```diff
@@ -251,39 +50,26 @@ func (observer *Observer) RunEvents(stopCtx context.Context, ready func()) error
 		defer wg.Done()
```

Comment:
idea for future cleanup.. it looks like RunEvents could be generic and we could have just windows/linux ring buffer specific code

Reply:
yes. I figured we could do that.